spec/core/spec_helper.rb

```diff
@@ -13,6 +13,7 @@
 require 'vcr'
 require 'logger'
 require 'stringio'
+require 'scrub_rb' if RUBY_VERSION < "2.1.0"
```

Comment:
My local JRuby 1.7.22 calls itself "2.0.0". The travis log says
```
$ ruby --version
jruby 1.7.19 (1.9.3p551) 2015-02-05 32f5af0 on Java HotSpot(TM) 64-Bit Server VM 1.7.0_76-b13 +jit [linux-amd64]
```
so that one would probably return "1.9.3".